darts/tests/models/forecasting/test_historical_forecasts.py

```diff
@@ -47,14 +47,28 @@
     ]
 
     models_reg_cov_cls_kwargs = [
-        # # target + past covariates
+        # target + past covariates
         (LinearRegressionModel, {"lags": 4, "lags_past_covariates": 6}, (6, 1)),
-        # target + past covariates + outputchunk > 1
+        # target + past covariates + outputchunk > 3
         (
             LinearRegressionModel,
             {"lags": 4, "lags_past_covariates": 6, "output_chunk_length": 4},
             (6, 4),
         ),
+        # target + future covariates
+        (LinearRegressionModel, {"lags": 4, "lags_future_covariates": [0, 1]}, (4, 2)),
+        # target + fut cov + output_chunk_length > 3
+        (
+            LinearRegressionModel,
+            {"lags": 2, "lags_future_covariates": [0, 1], "output_chunk_length": 6},
```

Comment:
This hasn't been adjusted yet after our discussion, right?

Reply:
Not yet, I need to run some experiments with `Timestamps`